### ISSUE TITLE
Error mapping for certifiedBy

### ISSUE THREAD

**Issue by mattgarrish** [daisy/ace#180]
As noted in the KB issue https://github.com/daisy/kb/issues/5 an error on a missing a11y:certifiedBy property maps to the schema.org metadata page, but it is not a schema.org property.

I've added a new page to the KB to cover the evaluation properties here: http://kb.daisy.org/publishing/docs/metadata/evaluation.html

**Comment by rdeltour** [comment]
> an error on a missing a11y:certifiedBy property maps to the schema.org metadata page

I don’t think that Ace actually reports any violation if `a11y:certifiedBy` is missing. It just lists it in the set of missing metadata in the metadata tab, but not as a violation… 🤔

**Comment by rdeltour** [comment]
@jenstroeger you said in daisy/kb#5

> The accessibility checker ace checks for the existence of the a11y:certifiedBy metadata property, and refers to the Accessibility Metadata page.

Can you please elaborate? (see my comment above: I don’t see that Ace reports any violation when `certifiedBy` is missing)

**Comment by danielweck** [comment]
Missing accessibility metadata and invalid values are mapped to http://kb.daisy.org/publishing/docs/metadata/schema.org/index.html
That's for `schema:accessMode`, `schema:accessibilityFeature`, `schema:accessibilityHazard`, `schema:accessibilitySummary`, `schema:accessModeSufficient` (`schema:accessibilityAPI` and `schema:accessibilityControl` now ignored as deprecated).

As Romain said, these are only reported as existing / missing, but without "violations": `a11y:certifierReport`, `a11y:certifiedBy`, `a11y:certifierCredential`, `dcterms:conformsTo`.
